.github/workflows/pack.yml

```diff
@@ -0,0 +1,152 @@
+                if ((.fixes | length) > 0) then
+                    (.fixes |
+                    map(\"* \" + .message + \"\n\") |
+                    . |= [\"### Fixes\n\"] + . |
+                    . |= . + [\"\n\"] |
+                    join(\"\"))
+                else (\"\")
+                end
+            ),
+            (
+                \$clFirstEntryLogs |
+                if ((.changes | length) > 0) then
+                    (.changes |
+                    map(\"* \" + .message + \"\n\") |
+                    . |= [\"### Changes\n\"] + . |
+                    . |= . + [\"\n\"] |
+                    join(\"\"))
+                else (\"\")
+                end
+            ),
+            (
+                \$clFirstEntryLogs |
+                if ((.removed | length) > 0) then
+                    (.removed |
+                    map(\"* \" + .message + \"\n\") |
+                    . |= [\"### Removed\n\"] + . |
+                    . |= . + [\"\n\"] |
+                    join(\"\"))
+                else (\"\")
+                end
+            )
+          " >> "$releaseBodyFile"
+          "${jqCommand}" -j ".[0].date |= \"$(date +"%Y/%m/%d")\"" "${changelogPath}" > "${tmpFile}" && mv "${tmpFile}" "${changelogPath}"
+          cd extension && zip -q -r "../torntools_${version}.zip" * && cd ..
+          [ ! -d "versions" ] && mkdir versions
+          mv "torntools_${version}.zip" ./versions/
+          "${jqCommand}" "
+            . |= 
+              [{
+                  \"version\": { \"major\": ${newVersionMajor}, \"minor\": ${newVersionMinor}, \"build\": ${newVersionBuild} },
+                  \"title\": \"${newVersionTitle}\",
+                  \"date\": false,
+                  \"logs\": {
+                      \"features\": [],
+                      \"fixes\": [],
+                      \"changes\": [],
+                      \"removed\": []
+                  }
+              }] +
+              ." "${changelogPath}" > "${tmpFile}" && mv "${tmpFile}" "${changelogPath}"
+          ghReleaseDateMarkdown="[![GitHub Release Date](https://img.shields.io/static/v1?label=RELEASE%20DATE\\\\&message=$(date "+%B, %-d$(DaySuffix). %Y" | "${jqCommand}" --raw-input --raw-output '. | @uri')\\\\&color=%23acea00\\\\&style=for-the-badge)](https://github.com/Mephiles/torntools_extension/releases/latest)<!---A pointer for bash-->"
+          awk "/.*A pointer for bash.*/ {sub(/.*/, \"${ghReleaseDateMarkdown}\")} {print}" "${readmePath}" > "${tmpFile}" && mv "${tmpFile}" "${readmePath}"
+          export version
+          echo "version=$version" >> "$GITHUB_ENV"
+      - name: Release
+        uses: ncipollo/release-action@v1
+        with:
+          artifacts: versions/torntools_${{ env.version }}.zip
+          name: "Release: v${{ env.version }}"
+          tag: "${{ env.version }}"
+          bodyFile: "./releaseBody.md"
+          token: ${{ secrets.GH_TOKEN }}
+      - name: Delete temporary files
```

Comment:
Shouldn't this also remove the zip file? It became redundant with the releases.

Reply:
I don't know that I should remove the zip file.
I will do that now.